sqlitecluster/SQLiteNode.cpp

```diff
@@ -2744,12 +2744,30 @@ SQLitePeer* SQLiteNode::_getPeerByID(uint64_t id) const {
     }
 }
 
-SQLitePeer* SQLiteNode::_getPeerByName(const string& name) const {
+bool comparePeers(SQLitePeer peer1, SQLitePeer peer2) {
```

Comment:
I believe these should be passed as `const` and by reference so you're not making a copy each time you call it.